src/posthog-core.ts

```diff
@@ -545,6 +548,14 @@ export class PostHog {
     }
 
     _onRemoteConfig(config: RemoteConfig) {
+        if (!(document && document.body)) {
```

Comment:
This was in decide before but should actually be here as this is the lower level call.